frontend/src/scenes/feature-flags/FeatureFlags.tsx

```diff
@@ -216,12 +219,16 @@ export function FeatureFlags(): JSX.Element {
                 }
             />
             {showActivityLog ? (
-                <Tabs defaultActiveKey="overview" destroyInactiveTabPane>
+                <Tabs activeKey={activeTab} destroyInactiveTabPane onChange={(t) => setActiveTab(t)}>
                     <Tabs.TabPane tab="Overview" key="overview">
                         <OverViewTab />
                     </Tabs.TabPane>
-                    <Tabs.TabPane tab="Activity" key="activity">
-                        <ActivityLog scope={ActivityScope.FEATURE_FLAG} describer={flagActivityDescriber} />
+                    <Tabs.TabPane tab="History" key="history">
+                        <ActivityLog
+                            scope={ActivityScope.FEATURE_FLAG}
+                            describer={flagActivityDescriber}
+                            startingPage={historyPage ?? 1}
```

Comment:
I wonder why this takes `startingPage` instead of `page` being fully controlled by the `activityLogLogic`?

Reply:
It was to keep the `ActivityLog` completely unaware of the URL

So, the containing scene knows about the URL and uses that to set a starting page. `ActivityLog` knows it is given a startingPage and knows about pagination, but doesn't know they are connected to the URL.

It feels more component-y that way??